NOTICE.txt

```diff
@@ -725,5 +725,35 @@ Legal Terms
 
     http://www.freetype.org
 
-
 --- end of FTL.TXT ---
+
+---
+
+This product contains a modified portion of 'marked', a full-featured markdown parser and compiler, written in JavaScript. Built for speed.
```

Comment:
Propose using the standard wording which covers the case where the library would get changed in future rather than trying to track its dirty state. Also "modified portion" can be interpreted broadly term and it's safer to say our use of the library is not perfectly identical to the original.